idseq_pipeline/commands/non_host_alignment_functions.py

```diff
@@ -828,7 +823,7 @@ def run_generate_unidentified_fasta(input_fa, output_fa):
     #TODO  remove annotated fasta intermediate file and replace > with : below
     subprocess.check_output("grep -A 1 '>NR::NT::' %s | sed '/^--$/d' > %s" % (input_fa, output_fa), shell=True)
     write_to_log("finished job")
-    execute_command("aws s3 cp --quiet %s %s/" % (output_fa, SAMPLE_S3_OUTPUT_PATH))
+    async_handler.launch_aws_cp(output_fa, SAMPLE_S3_OUTPUT_PATH + "/")
```

Comment:
We'll get in trouble when we start decorating these functions with @run_in_subprocess if they upload something to s3 at the very end.   If you are adding async handler inside such a function, better to pull the upload outside of the function.   Doesn't really apply to this specific instance, but something to think about.